webapp/src/app.tsx

```diff
@@ -149,6 +150,15 @@ const App = React.memo((): JSX.Element => {
                                             const loginUrl = `/login?r=${encodeURIComponent(redirectUrl)}`
                                             return <Redirect to={loginUrl}/>
                                         } else if (loggedIn === true) {
+                                            if (!localStorage.getItem('welcomePageViewed')) {
```

Comment:
I don't love this. It looks like, if I go to a board/workspace/card/view... (using this route) without having that localStorage variable set, I get "something" that is not what i expect, and if I close that "welcome" I get redirected to the wrong place. I would expect something like a `<Redirect>` here to the `welcome` and even with a `?next=whatever` query paramter. Also, that should affect other urls, like `/board/:boardId?/...` and the `/:boardId?/...`

Reply:
Thanks. Done